_maps/basemap.dm

```diff
@@ -13,7 +13,7 @@
 #include "map_files\Aetherwhisp\aetherwhisp.dmm"
 #include "map_files\SpaceSHIP\spaceship.dmm"
 #include "map_files\Trailblazer\trailblazer.dmm"
-#include "map_files\Scarab\scarab.dmm"
+#include "map_files\Scarab\Scarab.dmm"

```

Comment:
Good thinking, makes things appear a bit cleaner- even if it's not directly required.